Update deployment-guide.md

Ramyaharinarthini opened the pull request:
The below line has to be added in the document to install the AEM Extension version 0.2.2 or higher. Checked and confirmed with PG

NOTE: To install AEM Extension 0.2.2 or higher, the Latest version of AZ CLI must be installed.

Ramyaharinarthini committed:
Update deployment-guide.md

The below line has to be added in the document to install the AEM Extension version 0.2.2 or higher. Checked and confirmed with PG

NOTE: To install AEM Extension 0.2.2 or higher, the Latest version of AZ CLI must be installed.

MSSedusch commented on articles/virtual-machines/workloads/sap/deployment-guide.md:
```diff
@@ -982,8 +985,11 @@ To install the Azure Extension for SAP by using Azure CLI:
 
-   1. Install Azure CLI 2.0, as described in [Install Azure CLI 2.0][azure-cli-2].
-   1. Sign in with your Azure account:
+   1. Install Azure CLI 2.0, as described in [Install Azure CLI 2.0][azure-cli-2].   
```
I would rather put it like:
Install the lastest Azure CLI 2.0, as described in [Install Azure CLI 2.0][azure-cli-2]. Make sure to use at least version 2.19.1 or higher

MSSedusch commented on articles/virtual-machines/workloads/sap/deployment-guide.md:
```diff
@@ -982,8 +985,11 @@ To install the Azure Extension for SAP by using Azure CLI:
-   1. Sign in with your Azure account:
+   1. Install Azure CLI 2.0, as described in [Install Azure CLI 2.0][azure-cli-2].   
++
+NOTE: To install AEM Extension 0.2.2 or higher, the Latest version of AZ CLI must be installed.
```
move the note to step "Install Azure CLI AEM Extension"
"Install Azure CLI AEM Extension. Make sure to use at least version 0.2.2 or higher."

MSSedusch reviewed (changes requested):
could you also please make the same changes to the chapter regarding installing the new extension, i.e. line 1049 or 1055?

Ramyaharinarthini committed:
Update deployment-guide.md

updated the changes in line 1056 and 998

MSSedusch commented on articles/virtual-machines/workloads/sap/deployment-guide.md:
```diff
@@ -986,9 +986,7 @@ To install the Azure Extension for SAP by using Azure CLI:
 1. Install using Azure CLI 2.0
 
    1. Install Azure CLI 2.0, as described in [Install Azure CLI 2.0][azure-cli-2].   
```
'Install Azure CLI 2.0, as described in [Install Azure CLI 2.0][azure-cli-2]. Make sure to use at least version 2.19.1 or higher'

MSSedusch commented on articles/virtual-machines/workloads/sap/deployment-guide.md:
```diff
@@ -986,9 +986,7 @@ To install the Azure Extension for SAP by using Azure CLI:
 
    1. Install Azure CLI 2.0, as described in [Install Azure CLI 2.0][azure-cli-2].   
-+
-NOTE: To install AEM Extension 0.2.2 or higher, the Latest version of AZ CLI must be installed.
```
Please don't use NOTE section. IMHO this reduces the readability. Changing the text as described above should be good enough.

MSSedusch commented on articles/virtual-machines/workloads/sap/deployment-guide.md:
```diff
@@ -997,6 +995,9 @@ NOTE: To install AEM Extension 0.2.2 or higher, the Latest version of AZ CLI mus
 
    1. Install Azure CLI AEM Extension
```
1. Install Azure CLI AEM Extension. Make sure to use at least version 0.2.2 or higher.

MSSedusch commented on articles/virtual-machines/workloads/sap/deployment-guide.md:
```diff
@@ -997,6 +995,9 @@ NOTE: To install AEM Extension 0.2.2 or higher, the Latest version of AZ CLI mus
 
    1. Install Azure CLI AEM Extension
   
++ >[!Note]:Install the lastest Azure CLI 2.0, as described in [Install Azure CLI 2.0][azure-cli-2]. Make sure to use at least version 2.19.1 or higher +
```
Please don't use NOTE section. IMHO this reduces the readability. Changing the text as described above should be good enough.

MSSedusch commented on articles/virtual-machines/workloads/sap/deployment-guide.md:
```diff
@@ -1053,6 +1054,7 @@ The new VM Extension for SAP uses a Managed Identity assigned to the VM to acces
 1. Make sure to use SAP Host Agent 7.21 PL 47 or higher.
 1. Make sure to uninstall the current version of the VM Extension for SAP. It is not supported to install both versions of the VM Extension for SAP on the same virtual machine.
 1. Install Azure CLI 2.0, as described in [Install Azure CLI 2.0][azure-cli-2].
++  >[!Note]:Install the lastest Azure CLI 2.0, as described in [Install Azure CLI 2.0][azure-cli-2]. Make sure to use at least version 2.19.1 or higher +
```
Please don't use NOTE section. IMHO this reduces the readability. Changing the text as described above should be good enough.

MSSedusch commented on articles/virtual-machines/workloads/sap/deployment-guide.md:
```diff
@@ -1053,6 +1054,7 @@ The new VM Extension for SAP uses a Managed Identity assigned to the VM to acces
 1. Make sure to use SAP Host Agent 7.21 PL 47 or higher.
 1. Make sure to uninstall the current version of the VM Extension for SAP. It is not supported to install both versions of the VM Extension for SAP on the same virtual machine.
 1. Install Azure CLI 2.0, as described in [Install Azure CLI 2.0][azure-cli-2].
```
'Install Azure CLI 2.0, as described in [Install Azure CLI 2.0][azure-cli-2]. Make sure to use at least version 2.19.1 or higher'

MSSedusch reviewed (changes requested):
Could you please also change the version number in line 1065 from 0.2.0 to 0.2.2?

Ramyaharinarthini committed:
Update deployment-guide.md

Removed Note and changed lines 988,996,1053,1062 as per the comments mentioned.
988, 1053 as  +Install Azure CLI 2.0, as described in [Install Azure CLI 2.0][azure-cli-2]. Make sure to use at least version 2.19.1 or higher.+
998,1062 as +Install Azure CLI AEM Extension. Make sure to use at least version 0.2.2 or higher.+

Ramyaharinarthini committed:
Update deployment-guide.md

Added a space after line 996

Ramyaharinarthini committed:
Update deployment-guide.md

Committed changes in line 988, 996,1053,1062

Ramyaharinarthini committed:
Update deployment-guide.md

committed the changes in line 989,996,1053,1061

MSSedusch reviewed (approved):
lgtm. Thank you

ktoliver committed:
PR review edits
